nifi-registry/nifi-registry-core/pom.xml

```diff
@@ -130,7 +130,7 @@
             <dependency>
                 <groupId>com.google.guava</groupId>
                 <artifactId>guava</artifactId>
-                <version>28.0-jre</version>
+                <version>31.0.1-jre</version>
```

Comment:
The latest version of Guava is `31.1-jre`, which contains several updates to what is included in `31.0.1-jre`.